### PR TITLE
Add github workflow script and publish script for npm

### PR DESCRIPTION
**Pull request by purplenimbus**
Screen Shot 2023-06-15 at 2 59 51 AM

## What does this do
- Create's the raydial date picker component by the UI
- Add's the npm deploy script

## Why is it important
We want to use this date picker for our design system. Note that most of the functionality remains the same , all that was updated was the styles

### CODE REVIEW

File: src/components/Calendar/Days.tsx [hunk 1]
```diff
@@ -10,387 +10,371 @@ import { Period } from "../../types";
...
-            const fullDay = `${calendarData.date.year()}-${calendarData.date.month() + 1}-${day}`;
-            let className = "";
-
-            if (dayjs(fullDay).isSame(period.start) && dayjs(fullDay).isSame(period.end)) {
-                className = ` ${BG_COLOR["500"][primaryColor]} text-white font-medium rounded-full`;
-            } else if (dayjs(fullDay).isSame(period.start)) {
-                className = ` ${BG_COLOR["500"][primaryColor]} text-white font-medium ${
-                    dayjs(fullDay).isSame(dayHover) && !period.end
-                        ? "rounded-full"
-                        : "rounded-l-full"
```
Comment (purplenimbus) [inline src/components/Calendar/Days.tsx:68]: this is the main change. We are replacing the fully rounded buttons for rounded ones

File: .github/workflows/build-test-deploy.yml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/mirainc/raydiant-tailwind-component/issues"
+  },
+  "homepage": "https://github.com/mirainc/raydiant-tailwind-component#readme",
+  "dependencies": {
+    "npm-run-all": "^4.1.5",
+    "run-p": "^0.0.0"
+  }
+}
```
Comment (jschr) [inline .github/workflows/build-test-deploy.yml]:
```suggestion
```

Accidental copy-paste?

**Review by jschr — COMMENTED** [review]
Can you add a README? 

Should include any local development steps so others know how to make changes. Would be nice is there was a least a demo page or something so that a developer can make changes without linking to the Dashboard. Or maybe this is a good excuse to quickly try out Storybook for this one component.

File: .github/workflows/build-test-deploy.yml (hunk 2 above)
Comment (purplenimbus) [reply .github/workflows/build-test-deploy.yml]: Ye weird. I did it a 3am thats probably why

**Comment by purplenimbus** [comment]
> Can you add a README?
> 
> Should include any local development steps so others know how to make changes. Would be nice is there was a least a demo page or something so that a developer can make changes without linking to the Dashboard. Or maybe this is a good excuse to quickly try out Storybook for this one component.

We do have a demo page and a README file. The contents of the README are mostly that from the forked repo tho but it does contain instruction

**Comment by jschr** [comment]
> > Can you add a README?
> > Should include any local development steps so others know how to make changes. Would be nice is there was a least a demo page or something so that a developer can make changes without linking to the Dashboard. Or maybe this is a good excuse to quickly try out Storybook for this one component.
> 
> We do have a demo page and a README file. The contents of the README are mostly that from the forked repo tho but it does contain instruction

Ah didn't realize it was already in the repo. The README looks good enough for local dev.

**Review by jschr — APPROVED** [review]

**Pull request merged**